dotCMS/src/main/java/com/dotcms/rendering/velocity/services/ContentletLoader.java

```diff
@@ -0,0 +1,969 @@
+        sb.append(widgetCode);
+
+        // This is code is repeated because the bug GETTYS-268, the content
+        // variables were been overwritten
+        // by the parse inside the some of the content fields
+        // To edit the look, see
+        // WEB-INF/velocity/static/preview/content_controls.vtl
+
+        if (EDIT_MODE) {
+            sb.append("#set( $EDIT_CONTENT_PERMISSION=$EDIT_CONTENT_PERMISSION")
+                .append(identifier.getId())
+                .append(" )");
+        }
+
+        sb.append("#set( $CONTENT_INODE=\"")
+            .append(content.getInode())
+            .append("\" )");
+        sb.append("#set( $IDENTIFIER_INODE=\"")
+            .append(identifier.getId())
+            .append("\" )");
+
+        sb.append("#set( $ContentInode=\"")
+            .append(content.getInode())
+            .append("\" )");
+        sb.append("#set( $ContentIdentifier=\"")
+            .append(identifier.getId())
+            .append("\" )");
+        sb.append("#set( $ContentletTitle=\"")
+            .append(UtilMethods.espaceForVelocity(conAPI.getName(content, APILocator.getUserAPI()
+                .getSystemUser(), true)))
+            .append("\" )");
+        sb.append("#set( $ContentletStructure=\"")
+            .append(content.getContentTypeId())
+            .append("\" )");
+        sb.append("#set( $ContentletContentType=\"")
+            .append(content.getContentTypeId())
+            .append("\" )");
+        if (type.baseType() == BaseContentType.WIDGET) {
+            sb.append("#set( $isWidget= \"")
+                .append(true)
+                .append("\")");
+            if (type.name()
+                .equals(FormAPI.FORM_WIDGET_STRUCTURE_NAME_FIELD_NAME)) {
+                sb.append("#set($isFormWidget= \"")
+                    .append(true)
+                    .append("\")");
+            } else {
+                sb.append("#set($isFormWidget= \"")
+                    .append(false)
+                    .append("\")");
+            }
+        } else {
+            sb.append("#set($isWidget= \"")
+                .append(false)
+                .append("\")");
+        }
+
+        // close contentlet div
+        sb.append("#if($API_EDIT_MODE)")
+            .append("</div>")
+            .append("#end");
+
+        writeOutVelocity(filePath, sb.toString());
+
+
+        try {
+            result = new ByteArrayInputStream(sb.toString()
+                .getBytes("UTF-8"));
+        } catch (UnsupportedEncodingException e1) {
+            result = new ByteArrayInputStream(sb.toString()
+                .getBytes());
+            Logger.error(ContainerServices.class, e1.getMessage(), e1);
+        }
+        return result;
+    }
+
+    @SuppressWarnings("unchecked")
+    public List<Contentlet> sortContentlets(List<Contentlet> contentletList, String sortBy) {
+
+        Logger.debug(this.getClass(), "I'm ordering by " + sortBy);
+
+        if (sortBy.equals("tree_order")) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the "tree_order" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1132)